monai/inferers/utils.py

```diff
@@ -307,9 +307,9 @@ def sliding_window_inference(
             output_image_list[ss] = output_i[(slice(None), slice(None), *final_slicing)]
 
     final_output = _pack_struct(output_image_list, dict_keys)
-    final_output = convert_to_dst_type(final_output, inputs, device=device)[0]  # type: ignore
```

Comment:
If `temp_meta` is None, seems you deleted the logic to convert `final_output` to the target `device` and `inputs type`?

Thanks.

Reply:
(addressed by https://github.com/Project-MONAI/MONAI/pull/6325/commits/04da4391e661db83f31e956623beaf6ae383e122)